src/js/monkey.js

```diff
@@ -78,7 +78,7 @@ window.Monkey = module.exports = (function () {
     }
 
     promise = promise
-      .then(Monkey._generateUrls(options.preload))
+      .then(Monkey._generateUrls(options.preload, options))
```

Comment:
Isn't it being used here?

https://github.com/Lostmyname/monkey/blob/master/src/js/steps/generateUrls.js#L49